lib/topological_inventory/ansible_tower/collector.rb

```diff
@@ -55,7 +55,7 @@ def service_catalog_entity_types
 
     # Connection to endpoint (for each entity type the same)
     def connection_for_entity_type(_entity_type)
-      connection_manager.connect(tower_hostname, tower_user, tower_passwd)
+      connection_manager.connect(:base_url => tower_hostname, :username => tower_user, :password => tower_passwd)
```

Comment:
So I see we have a receptor collector now - maybe we should rename this to cloud collector or something like that?

Reply:
that's a good idea!

Reply:
@lindgrenj6 so I renamed collector to `Cloud::Collector` and Connection to `Cloud::Connection`. The PR shows them as new classes but there are no changes except the `connection_manager.connect()`